src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/Rules/PropertyPages/BuildPropertyPage.xaml

```diff
@@ -173,11 +173,68 @@
                DisplayName="Embedded in DLL/EXE, portable across platforms" />
   </EnumProperty>
 
+  <BoolProperty Name="WarningLevelOverridden"
+                DisplayName="WarningLevelOverridden"
+                Category="ErrorsAndWarnings"
+                ReadOnly="True">
+    <BoolProperty.DataSource>
+      <DataSource HasConfigurationCondition="True"
+                  Persistence="ProjectFileWithInterception" />
+    </BoolProperty.DataSource>
+    <BoolProperty.Metadata>
+      <NameValuePair Name="VisibilityCondition" Value="false" />
+    </BoolProperty.Metadata>
```

Comment:
Setting `Visible="False"` on the property directly is cleaner and runs slightly faster.

Reply:
Right. Forgot about that one!